src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public int getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        System.out.println(color.getRandomColor());
+        Random random = new Random();
+        int number = random.nextInt(100);
+        return number;
```

Comment:
The method name `getRandomBall` suggests that it should return a Ball object, but it currently returns an integer. According to the method's responsibility, it should create a new Ball instance using the random color from the ColorSupplier and a random number for the ball, and then return this Ball object.